src/main/java/org/radarbase/appserver/service/questionnaire/protocol/SimpleReminderHandler.java

```diff
@@ -45,7 +45,8 @@ public List<Notification> generateReminders(List<Task> tasks, Assessment assessm
                         return notification;
                     });
                         }
-                ).collect(Collectors.toList());
+                )
+                .filter(notification -> (Instant.now().isBefore(notification.getScheduledTime()))).collect(Collectors.toList());
```

Comment:
Should we still check the ttl or completion window here. the reminder is valid till the task is valid.

Reply:
Ah yes I updated this on the notification handler but didn't for the reminder handler. I've updated it now.